src/api/pros.py

```diff
@@ -0,0 +1,99 @@
+from flask import Blueprint, jsonify, request, abort
+import uuid
+import datetime
+import json
+from utils import load_json, save_json
+
+pros = Blueprint('pros', __name__, url_prefix='/pros')
+
+
+@pros.route('/', methods=['GET'])
+def list_pros():
+    """
+    Fetches list of professionals
+    """
+    db = load_json('./src/api/db.json')
+    if not db:
+        abort(500)
+
+    return jsonify(db), 200
+
+
+@pros.route('/', methods=['POST'])
+def create_pro():
+    """
+    Adds a new professional to directory
+    """
+    data = request.get_json()
+    if not data:
+        abort(400)
```

Comment:
Can we have description as to why 400? Eg: `{ 'error': 'Empty request data' }` or something along those lines.

Reply:
Done